### DISCUSSION TITLE
Interactive shell REPL

### DISCUSSION THREAD

**Discussion by dcrystalj** [adonisjs/core#1379] [Ideas]
Is there an easy way to run node interactive shell where I can import `import Database from '@ioc:Adonis/Lucid/Database'` on the fly ?

In laravel there is Tinker https://laravel.com/docs/7.x/artisan#tinker
Is there anything similar in adonisjs for node shell ?

**Comment by thetutlage** [comment]
Duplicate of https://github.com/adonisjs/core/discussions/1374
